frontend/sw360-portlet/src/main/resources/META-INF/resources/html/projects/includes/projects/clearingRequest.jspf

```diff
@@ -67,9 +69,18 @@
                             <label for="requestedClearingDate" class="mandatory"><liferay-ui:message key="preferred.clearing.date" />:</label>
                             <input class="datepicker form-control" id="requestedClearingDate" type="text" pattern="\d{4}-\d{2}-\d{2}" placeholder="<liferay-ui:message key='preferred.clearing.date.yyyy.mm.dd' />" required/>
                             <div class="invalid-feedback">
-                                <liferay-ui:message key="date.should.be.in.valid.format" /> & <liferay-ui:message key="greater.than.7.days.from.today" />
+                                <liferay-ui:message key="date.should.be.valid" />!
```

Comment:
May be this needed inside the angular tag, or else please remove.

Reply:
removed exclamation.